engine/table/src/main/java/io/deephaven/engine/table/impl/select/analyzers/SelectAndViewAnalyzer.java

```diff
@@ -458,6 +777,15 @@ public UpdateHelper(RowSet parentRowSet, TableUpdate upstream) {
             this.upstream = upstream;
         }
 
+        TableUpdate resultKeySpaceUpdate() {
+            if (upstreamInResultSpace == null) {
```

Comment:
Maybe a comment:
// This means we are flattening a dummy update from the parent key space to the destination key space in a static select.